IntegrationAdaptersTests/ActionsAndBenefitsTests.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Moq;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using IntegrationAdapters;
+using IntegrationAdapters.Models;
+using IntegrationAdapters.Repositories;
+using Xunit;
+using IntegrationAdapters.Services;
+
+namespace IntegrationAdaptersTests
+{
+    public class ActionsAndBenefitsTests
+    {
+        public Mock<IActionsAndBenefitsRepository> mock;
+        public Guid guid = new Guid("0e89a9a7-2722-45bb-a3de-7832131dcfe3");
+        public Guid guid2 = new Guid("a69b4d8e-55ce-4cdf-bfa6-7462e085a909");
+
+        [Fact]
+        public void Find_existing_published_message()
+        {
+            mock = new Mock<IActionsAndBenefitsRepository>();
+            mock.Setup(expression: t => t.GetActionAndBenefitMessegeByID(It.IsAny<Guid>())).Returns( new ActionAndBenefitMessage { ActionID = guid, PharmacyName = "Jankovic", Text = "50% off brufen", DateFrom = new DateTime(2020,5,21) ,DateTo = new DateTime(2020,6,21)} );
```

Comment:
To continue on the previous comment, just instantiate this ActionAndBenefitsMessage in the field and use it in all the tests